src/NovelDebugService.cpp

```diff
@@ -25,8 +25,8 @@ messageCallback(GLenum source,
   glDebugMessageCallback(messageCallback, 0);*/
 
 #include <limits>
-#include "NovelDebugService.h"
-#include "NovelRunner.h"
+#include "../include/NovelDebugService.h"
+#include "../include/NovelRunner.h"
```

Comment:
it might be a good idea to just change these to `#include <NovelRT.h>` while you're here

Reply:
(for all the cpp files)

Reply:
Only for C++ files? Is there a reason for that?

Reply:
For the header files, it depends on if we want to enforce users only reference `NovelRT.h` (in which case they should have no includes and might want to error if `NOVELRT_H` isn't defined) or if you want them to be "self-contained"

Reply:
I see....well, for end-users I'd probably only want them using `NovelRT.h` in their projects, but what do you think is better?

Reply:
I think that's probably a reasonable starting position and we can change later if needed

Reply:
Alright, so do I stick with your original request for that and just check for the NovelRT define?

Reply:
Yes, that should work

Reply:
Got it!

Reply:
Since this branch is now working again (as of this commit) and this structure has been implemented, I am marking this as resolved before I forget.